Lecture4/src/test/java/CalcUnitTests.java

```diff
@@ -0,0 +1,142 @@
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class CalcUnitTests {
+	private double op1;
```

Comment:
all tests should be independent
so do not use this private fields
you already have all what you need it test parameters

Reply:
fixed